var/spack/repos/builtin/packages/wrf/package.py

```diff
@@ -15,132 +60,269 @@ class Wrf(Package):
+    depends_on("parallel-netcdf", when="+pnetcdf")
+    depends_on("netcdf-c")
+    depends_on("netcdf-fortran")
+    depends_on("jasper")
+    depends_on("libpng")
+    depends_on("zlib")
+    depends_on("perl")
     # not sure if +fortran is required, but seems like a good idea
-    depends_on('hdf5+fortran+hl+mpi')
+    depends_on("hdf5+fortran+hl+mpi")
     # build script use csh
-    depends_on('tcsh', type=('build'))
+    depends_on("tcsh", type=("build"))
     # time is not installed on all systems b/c bash provides it
     # this fixes that for csh install scripts
-    depends_on('time', type=('build'))
-    depends_on('m4', type='build')
-    depends_on('libtool', type='build')
-    phases = ['configure', 'build', 'install']
+    depends_on("time", type=("build"))
+    depends_on("m4", type="build")
+    depends_on("libtool", type="build")
+    phases = ["configure", "build", "install"]
 
     def setup_build_environment(self, env):
-        env.set('NETCDF', self.spec['netcdf-c'].prefix)
-        if '+pnetcdf' in self.spec:
-            env.set('PNETCDF', self.spec['parallel-netcdf'].prefix)
+        env.set("NETCDF", self.spec["netcdf-c"].prefix)
+        if "+pnetcdf" in self.spec:
+            env.set("PNETCDF", self.spec["parallel-netcdf"].prefix)
         # This gets used via the applied patch files
-        env.set('NETCDFF', self.spec['netcdf-fortran'].prefix)
-        env.set('PHDF5', self.spec['hdf5'].prefix)
-        env.set('JASPERINC', self.spec['jasper'].prefix.include)
-        env.set('JASPERLIB', self.spec['jasper'].prefix.lib)
+        env.set("NETCDFF", self.spec["netcdf-fortran"].prefix)
+        env.set("PHDF5", self.spec["hdf5"].prefix)
+        env.set("JASPERINC", self.spec["jasper"].prefix.include)
+        env.set("JASPERLIB", self.spec["jasper"].prefix.lib)
 
-        if self.spec.satisfies('%gcc@10:'):
-            args = '-w -O2 -fallow-argument-mismatch -fallow-invalid-boz'
-            env.set('FCFLAGS', args)
-            env.set('FFLAGS', args)
+        if self.spec.satisfies("%gcc@10:"):
+            args = "-w -O2 -fallow-argument-mismatch -fallow-invalid-boz"
+            env.set("FCFLAGS", args)
+            env.set("FFLAGS", args)
 
     def patch(self):
         # Let's not assume csh is intalled in bin
-        files = glob.glob('*.csh')
+        files = glob.glob("*.csh")
+
+        filter_file("^#!/bin/csh -f", "#!/usr/bin/env csh", *files)
+        filter_file("^#!/bin/csh", "#!/usr/bin/env csh", *files)
+
+    def answer_configure_question(self, outputbuf):
 
-        filter_file('^#!/bin/csh -f', '#!/usr/bin/env csh', *files)
-        filter_file('^#!/bin/csh', '#!/usr/bin/env csh', *files)
+        # Platform options question:
+        if "Please select from among the following" in outputbuf:
+            options = collect_platform_options(outputbuf)
+            comp_pair = "%s/%s" % (
+                basename(self.compiler.fc),
+                basename(self.compiler.cc),
```

Comment:
`self.compiler.cc` and friends are the actual compilers, generally we want to use Spack's compiler wrappers (`spack_cc`) so that we automatically inject the appropriate linking args. Does it not work if you use Spack's compiler wrappers?

Reply:
Here I am picking up the actual name of the compiler binary to compare with the options presented by the interactive configure script.  It doesn't actually inject any paths.